docs/usage/graphql.md

```diff
@@ -2,11 +2,21 @@
 
 Once OAuth is complete, we can use the library's `GraphqlClient` to make requests to the GraphQL Admin API in a similar way. To do that, create an instance of `GraphqlClient` using the current shop URL and session `accessToken` in your app's endpoint.
 
+The `GraphQLClient`'s main method is `query`, which accepts a `GraphQLParams` object as its argument. `GraphQLParams` only requires the `data` parameter, but also optionally accepts `query`, `extraHeaders`, and `tries`:
+
+| Parameter      | Type                                | Notes                                                                                 | Required? | Default Value |
```

Comment:
I think a good rule is to always put the "Notes" column last, since it's often the largest chunk and in some contexts might push more critical information (like "Required?") out of the user's view

Reply:
Oh that's a really good point, thank you! I'll shift that over.

Reply:
@paulomarg Yeah I feel the same way. Is it obvious enough to just end the param name in a `?` to mark it as optional? So few of the params are required it feels like a waste of table space. I actually feel the same way about the `Default Value` column, but want to communicate the default for `tries` in an obvious way.